src/content/docs/apm/agents/nodejs-agent/api-guides/nodejs-agent-api.mdx

```diff
@@ -176,6 +176,10 @@ Use these API calls to [expand your instrumentation with custom instrumentation]
     id="instrumentLoadedModule"
     title={<InlineCode>newrelic.instrumentLoadedModule(moduleName, moduleInstance)</InlineCode>}
   >
+    <Callout variant="important">
+      This method is not supported in ES module applications.
```

Comment:
Not sure if we want to call this out but it's technically not needed in ESM. Maybe I'm wrong but due to how the experimental loader registers instrumentation I don't think we have the same issues we did with bootstrapping agent in cjs